bcipy/signal_model/mach_learning/classifier/function_classifier.py

```diff
@@ -164,7 +165,7 @@ def get_prob(self, x):
                 evidence = np.dot(zero_mean,
                                   np.dot(self.inv_reg_cov_i[i],zero_mean))
 
-                neg_log_l[s][i] = -evidence + np.log(self.prior_i[i])
+                neg_log_l[s][i] = -.5*evidence + np.log(self.prior_i[i])
```

Comment:
Should this be a variable / removed? this would change existing functionality would it not?

Reply:
I had already answered your question.

![image](https://user-images.githubusercontent.com/30954743/41723650-9935f458-7539-11e8-8bc2-48142d6f3477.png)

This should not be a variable. This should not be removed. This would change existing functionality. If you need confirmation, we can ask other people their opinion as well.

Reply:
Because this is code used by the other offline analysis method, not the new one you've made, changing this would change existing functionality. I'm not saying you're wrong. Was this an oversight on the old PR?